podman-compose.yml

```diff
@@ -5,25 +5,34 @@ networks:
     external: false
 
 services:
-  local.altinn.cloud:
+  localtest_loadbalancer:
     container_name: localtest-loadbalancer
     image: nginx:alpine-perl
     restart: always
     networks:
-      - altinntestlocal_network
+      altinntestlocal_network:
+        aliases:
+          - ${TEST_DOMAIN:-local.altinn.cloud}
     ports:
       - "${ALTINN3LOCAL_PORT:-8080}:80"
     environment:
       - NGINX_HOST=localhost
       - NGINX_PORT=80
       - TEST_DOMAIN=${TEST_DOMAIN:-local.altinn.cloud}
+      - HOST_DOMAIN=host.docker.internal
       - INTERNAL_DOMAIN=host.containers.internal
-      - ALTINN3LOCAL_PORT=${ALTINN3LOCAL_PORT:-80}
+      - ALTINN3LOCAL_PORT=${ALTINN3LOCAL_PORT:-8080}
```

Comment:
Can you remind me why you changed from 80 to 8080?

Reply:
Podman rootless machines (machines without complete admin rights) cannot bind to ports <1024.
Modern linux systems regards ports <1024 as privileged ports and require root(admin) rights to be used. I think we should move away from requiring developers to running localtest with root

Reply:
Does this port change work for pdf generation?

Apps send an url to chromium which needs to be fully qualified, and currently I'd assume this will break apps.

In addition frontend seems to be using port:8080 for dev, which will conflict with running load balancer on that port
https://github.com/Altinn/app-localtest/blob/c55ae94824e1eadc51fed91e0ebb4c60cf4ec0ee/src/Controllers/FrontendVersionController.cs#L42-L43

Reply:
Port works for pdf generation without changes to existing apps. Applications sends a request to the chromium cotnainer with port 80 and internally in the docker/podman-network the port is still 80.

Had a discussion about frontend using 8080. It's quite common to use 8080 locally for services that normally would use 80, but can't due to restrictions on running privileged ports.
Fontend dev port isn't something users normally interact with directly so I think we rather should change fontends port instead of localtest.

Reply:
Made the port configurable for localtest so it can be changed more easily

Reply:
Switched to port 8000, localtest scans for local dev-server running app-frontend-react in the range 8080..8090.
PDF is working as before without changes to apps